modules/carto/src/layers/schema/carto-vector-tile-loader.ts

```diff
@@ -29,4 +31,4 @@ function parseCartoVectorTile(
   return tile as unknown as BinaryFeatures;
 }
 
-export default CartoVectorTileLoader;
+export default createWorkerLoader(CartoVectorTileLoader);
```

Comment:
Nit: I am admittedly biased towards typing things out rather then abbreviating things. but personally I feel this helper is a step too far in the tradeoff between saving typing and making code as readable as possible: 
- You don't save a lot of lines of code, and this is helper name is so similar to `createLoaderWorker` that things just got confusing for me :)
- Having to chase down this helper to see what is going on is not helpful
- Workers are such a complicated subject that keeping code super readable is worthwhile IMHO.